tool/travis.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Fast fail the script on failures.   
+set -e
+
+pub run test -p content-shell
```

Comment:
You can also test on firefox on travis. See https://gist.github.com/a14n/441101c997ee8f935d9c#file-travis-yml-L10